include/sys/zap.h

```diff
@@ -307,7 +307,7 @@ int zap_join(objset_t *os, uint64_t fromobj, uint64_t intoobj, dmu_tx_t *tx);
 
 /* Same as zap_join, but set the values to 'value'. */
 int zap_join_key(objset_t *os, uint64_t fromobj, uint64_t intoobj,
-    uint64_t value, dmu_tx_t *tx, boolean_t exists_ok);
```

Comment:
This looks like a revert to an earlier state of this code, but it also looks like zap_join_key was only ever used in dsl_scan.c and now is not; is it worth removing this function?